wallet/wallet.c

```diff
@@ -47,14 +47,11 @@ struct channel_state_param {
 };
 
 /* possible channels states */
-struct channel_state_param channel_state_params[] = {
+static struct channel_state_param channel_state_params[] = {
```

Comment:
it'd be better to fold this change in with the commit that originally made the line.